pymc3/tests/test_distributions.py

```diff
@@ -284,6 +284,10 @@ def mvt_logpdf(value, nu, Sigma, mu=0):
     return log_pdf
 
 
+def AR1_logpdf(value, k, tau_e):
+    return (sp.norm(loc=0,scale=1/np.sqrt(tau_e)).logpdf(value[0]) +
```

Comment:
I dont think we should use `statsmodels` in the test. I am thinking more rewriting our AR1 logp similar to the (exact) unconditional maximum likelihood as in `statsmodels`. We should take this opportunity to extend our AR1 model into AR(n).